cudax/include/cuda/experimental/__execution/cpos.cuh

```diff
@@ -131,15 +131,39 @@ struct start_t
   }
 };
 
+template <class _Sndr, class _Rcvr>
+_CCCL_CONCEPT __has_connect_mbr = //
+  _CCCL_REQUIRES_EXPR((_Sndr, _Rcvr), _Sndr& __sndr, _Rcvr& __rcvr) //
```

Comment:
We could try and tell clang-format that `_CCCL_REQUIRES_EXPR` is a `IfStatementMacro`